packages/backend/src/services/CsvService/CsvService.ts

```diff
@@ -186,19 +214,17 @@ export class CsvService {
         stringifier.write(csvHeader);
 
         // Increasing CHUNK_SIZE increases memory usage, but increases speed of CSV generation
+        // This method should be running on it's own thread on schedulerWorker
```

Comment:
It's ok to block the workers! This is what they are for! We can scale the number of workers instead of the threads when there's a heavy load. That way we don't need to copy memory etc.

Reply:
Yes ! exactly. That's why I've removed the WorkerThread implementation that was running this code on a separate thread.